airflow/www/static/js/ti_log.js

```diff
@@ -102,9 +102,8 @@ function autoTailingLog(tryNumber, metadata = null, autoTailing = false) {
 
       // Detect urls and log timestamps
       const urlRegex = /http(s)?:\/\/[\w.-]+(\.?:[\w.-]+)*([/?#][\w\-._~:/?#[\]@!$&'()*+,;=.%]+)?/g;
-      const dateRegex = /(\d{4}[./-]\d{2}[./-]\d{2} \d{2}:\d{2}:\d{2})((,\d{3})|([+-]\d{4} \d{3}ms))/g;
-      // above regex is a kind of duplication of 'dateRegex'
-      // in airflow/www/static/js/grid/details/content/taskinstance/Logs/utils.js
+      const dateRegex = /\d{4}[./-]\d{2}[./-]\d{2} \d{2}:\d{2}:\d{2},\d{3}/g;
+      const iso8601Regex = /\d{4}[./-]\d{2}[./-]\d{2}T\d{2}:\d{2}:\d{2}.\d{3}[+-]\d{4}/g;
```

Comment:
The regular expression on the `utils.ts` side is `const regExp = /\[(.*?)\] \{(.*?)\}/;`, which finds the string between square brackets instead of finding the time string.
moments.js understands the ISO 8601 format, so we don't have to worry about this.